### PR TITLE
Shayan/76595/div100vh container typescript migration

### PR DESCRIPTION
**Pull request by shayan-deriv**
## Changes:

Please include a summary of the change and which issue is fixed below:
-   ...

## When you need to add unit test

-   [ ] If this change disrupt current flow
-   [ ] If this change is adding new flow

## When you need to add integration test

-   [ ] If components from external libraries are being used to define the flow, e.g. @deriv/components
-   [ ] If it relies on a very specific set of props with no default behavior for the current component.

## Test coverage checklist (for reviewer)

-   [ ] Ensure utility / function has a test case 
-   [ ] Ensure all the tests are passing

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: packages/components/src/components/div100vh-container/div100vh-container.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+const Div100vhContainer = ({
+    children,
+    className,
+    is_bypassed = false, // to bypass usage of component altogether, pass it
+    is_disabled = false, // To manually remove use100vh() calculation and revert to default browser calculation pass it
+    id,
+    height_offset,
+    max_autoheight_offset,
+}: TDiv100vhContainerProps) => {
```
Comment (yauheni-deriv) [inline packages/components/src/components/div100vh-container/div100vh-container.tsx]: I think we can remove children from types and add here following: PropsWithChildren<TDiv100vhContainerProps>

you can have a look on this article: https://www.newline.co/@bespoyasov/how-to-define-props-with-children-in-react-typescript-app--56bd18be

Comment (iman-fs) [reply packages/components/src/components/div100vh-container/div100vh-container.tsx:23]: yes I saw that recently in your PR. will change it :+1:

Comment (shayan-deriv) [reply packages/components/src/components/div100vh-container/div100vh-container.tsx]: @yauheni-kryzhyk-deriv but in my component Children is not optional. when we use `PropsWithChildren`, TS consider the children as optional.

Comment (shayan-deriv) [reply packages/components/src/components/div100vh-container/div100vh-container.tsx]: I checked it. It's safe to apply your sugestion. thanks @yauheni-kryzhyk-deriv

**Review by yauheni-deriv — APPROVED** [review]

**Review by jim-deriv — APPROVED** [review]

**Pull request merged**